samples/bluetooth/hci_ipc/nrf5340_cpunet_bis-bt_ll_sw_split.conf

```diff
@@ -40,7 +40,11 @@ CONFIG_BT_ISO_PERIPHERAL=n
 
 # ISO Streams
 CONFIG_BT_ISO_MAX_CHAN=4
-CONFIG_BT_ISO_TX_BUF_COUNT=1
+# In theory, CONFIG_BT_ISO_TX_BUF_COUNT=1, should be sufficient but this count
+# is used in the context of IPC which falls into a "Newton's Cradle" effect
+# where probably (CONFIG_BT_CTLR_ISO_TX_BUFFERS - CONFIG_BT_ISO_TX_BUF_COUNT)
+# buffers get throttled. Hence, always have the value equal or greater.
+CONFIG_BT_ISO_TX_BUF_COUNT=12
```

Comment:
Could this be further explained in a GH issue?

This isn't something I understand as it is written here